stage1/init/init.go

```diff
@@ -577,16 +577,17 @@ func stage1() int {
 		return 1
 	}
 
-	// create a separate mount namespace so the cgroup filesystems
-	// are unmounted when exiting the pod
+	// Creates a separate mount namespace so the cgroup filesystems
```

Comment:
It is subtly clearer -- and hard to explain why -- in the former, imperative style.

The new descriptive style seems to promote them to godoc function descriptions, which they are not.

Making them sentences doesn't seem a huge addition of value, and isn't done with great consistency here anyway. -1 with the exception of the spelling/grammar cleanup, noted at that line.